### PR TITLE
chore(Examples): add Vite example

### PR DESCRIPTION
**Pull request by pladaria**
Vite project bootstrapped with `yarn create vite` and updated with a simple Mistica page

image

### CODE REVIEW

File: examples/vite/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# React + TypeScript + Vite + Mistica
+
+This template provides a minimal setup to get React working in Vite with HMR, some ESLint rules and using
+Mistica Design System.
```
Comment (pladaria) [inline examples/vite/README.md:4]: This is the autogenerated README. I've modified this

File: examples/vite/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  `plugin:@typescript-eslint/strict-type-checked`
+- Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
+- Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add
+  `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+
+## Notes
+
+The `package.json` has a `resolutions` property to force the `@swc/core` version to workarond
+[this issue](https://github.com/swc-project/swc/issues/8996#issuecomment-2137309969). Once fixed you can
+remove it.
```
Comment (pladaria) [inline examples/vite/README.md:42]: and this

File: examples/vite/src/app.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                        message: JSON.stringify(formData, null, 2),
+                    })
+                }
+            >
+                <Box padding={16}>
+                    <Stack space={16}>
+                        <img height={40} src="/vite.svg" alt="Vite Logo" />
+                        <TextField name="name" label="Name" autoComplete="name" />
+                        <EmailField name="email" label="E-mail" autoComplete="email" />
+                        <ButtonLayout>
```
Comment (atabel) [inline examples/vite/src/app.tsx:30]: `children` is deprecated in `ButtonLayout`, please use `primaryButton` prop instead

**Review by atabel — APPROVED** [review]

**Review by marcoskolodny — APPROVED** [review]

**Pull request merged**